frontend/components/CellOutput.js

```diff
@@ -401,7 +401,10 @@ export let highlight = (code_element, language) => {
                 code_element.classList.add("cm-s-default")
             },
             {
-                path: (mode) => `https://cdn.jsdelivr.net/npm/codemirror@5.60.0/mode/${mode}/${mode}.min.js`,
+                // Commented out since 
+                // 1) not needed since this is manually loaded in editor.html already
+                // 2) it is loaded w/o integrity verification which we don't want
+                // path: (mode) => `https://cdn.jsdelivr.net/npm/codemirror@5.60.0/mode/${mode}/${mode}.min.js`,
```

Comment:
This code will lazy-load codemirror modes for new programming languages, to highlight code snippets in HTML output. E.g. a `python` code block will trigger loading the Python mode to highlight that block.

It will require some extra work to make this work with the SRI-only restriction. @schneiderfelipe might have ideas

Reply:
Note that we are planning to switch to codemirror 6 soon, which would require rewriting this mechanism anyways. We might need to switch to a dedicated highlighting library, which would come preloaded with many programming languages. 

We could look into https://highlightjs.org/ as an alternative to this lazy-loading trick.

Reply:
Ah, I see. I just added SRI verification also for these lazy-loaded files. This requires manually adding the hashes for all mode files we want to support to the `modeSRIhashes` dictionary (see newest commit). I could write a small script to auto-generate all hashes from this directory listing

https://cdn.jsdelivr.net/npm/codemirror@5.60.0/mode/

This should not be too much effort, but I haven't done that yet. If you think this way of doing things is okay, I can write this script. If you decide to switch to a different highlighting library right away, this will not be necessary of course.